public/search.php

```diff
@@ -1,137 +1,48 @@
+?>
+
+<section class="search-page-form row">
+    <div class="col-lg-2 col-12"></div>
+    <div class="col-lg-8 col-12">
+        <?php require_once('includes/dynamic-select-form.php'); ?>
     </div>
-
-    <hr>
+    <div class="col-lg-2 col-12"></div>
 </section>
-<section class="search-ready">
-    <h3 class="search-ready-heading">Know when your home would be ready</h3>
-    <p class="search-ready-text">Fill in the details below to  know when your home would be ready</p>
-
-    <div class="row search-ready-investments">
-        <div class="col-lg-3 search-ready-step investment-frequency">
-            <h5 class="search-ready-step-heading">Investment Frequency</h5>
-            <p class="search-ready-step-text">Select one of the options below</p>
-
-            <div class="btn-group-toggle investment-options row" data-toggle="buttons">
-                <label class="col-lg-12 col-sm-3 btn btn-secondary option-btn">
-                    <input type="radio" name="investment-frequency" id="option1" autocomplete="off"> Monthly
-                </label>
-
-                <label class="col-lg-12 col-sm-3 btn btn-secondary option-btn option-btn-quaterly active">
-                    <input type="radio" name="investment-frequency" id="option2" autocomplete="off" checked> Quarterly
-                </label>
-
-                <label class="col-lg-12 col-sm-3 btn btn-secondary option-btn">
-                    <input type="radio" name="investment-frequency" id="option3" autocomplete="off"> Voluntarily
-                </label>
-            </div>
-        </div>
 
-        <div class="col-lg-3 search-ready-step investment-initial">
-            <h5 class="search-ready-step-heading">Initial investment amount</h5>
-            <p class="search-ready-step-text">Use the scroll bar or type in <br> the desired amount</p>
+<?php
+if (isset($_GET["index-form-search"])) {
+    $state_id = $mysqli->real_escape_string($_GET['state']);
+    $area_id = $mysqli->real_escape_string($_GET['area']);
+    $house_type = $mysqli->real_escape_string($_GET['type']);
+
+    $sql_query = "SELECT houses.type, houses.price, area.area_name, states.state_name, AVG(price) as average_price, MIN(price) as minimum_price, MAX(price) as maximum_price FROM houses INNER JOIN area INNER JOIN states WHERE houses.area_id = '$area_id' AND area.area_id = houses.area_id AND states.state_id = houses.state_id AND houses.type='$house_type'";
+    $result = mysqli_query($mysqli,$sql_query);
+    $no_of_results = $result->num_rows;
+        
+    if ($no_of_results > 0) {
+        $row = mysqli_fetch_array($result, MYSQLI_ASSOC);
+        $state = $row['state_name'];
+        $area = $row['area_name'];
+        $type = $row['type'];
+        $avg_price = number_format($row['average_price']);
+        $min_price = number_format($row['minimum_price']);
+        $max_price = number_format($row['maximum_price']);
+        $search_view = 'includes/search-valid.php';
+    } else {
```

Comment:
I don't think the else part is needed since the default value of $search_view is `includes/search-invalid.php`